plugins/services/src/js/containers/service-configuration/__tests__/ServiceConfigurationContainer-test.js

```diff
@@ -47,17 +69,5 @@ describe("ServiceConfigurationContainer", function() {
 
       expect(serviceSpecView).toBeDefined();
     });
-
-    it("doesn't pass down onEditClick to ServiceConfigDisplay", function() {
```

Comment:
Removing because we are not using `onEditClick` in `ServiceStorageConfigSection.js` anymore